src/modules/tools/zprobe/ZGridStrategy.cpp

```diff
@@ -327,24 +332,19 @@ bool ZGridStrategy::saveGrid() //std::string *args)
 
 }
 
-bool ZGridStrategy::loadGrid()
+bool ZGridStrategy::loadGrid(std::string args)
```

Comment:
It may be inefficient, but the pointers are messing with my head today.

At least this works. ;-)